### PR TITLE
Query service template history and query request ocl by changeId

### PR DESCRIPTION
**Pull request by baixinsui**
**Fixed https://github.com/eclipse-xpanse/xpanse/issues/2106**

**New service `getServiceTemplateHistoryById`:**
chrome_4vUeJ8Yi7Q
Call this service with serviceTemplateId well:
image
Call this service throw exception:
image

**New service `getRequestedServiceTemplateByChangeId`:**
image
Call this service with changeId well:
image
Call this service throw exception:
image

### CODE REVIEW

**Comment by Alice1319** [comment]
LGTM

**Comment by wuzhuanhong** [comment]
LGTM

**Comment by profoundwu** [comment]
LGTM

**Comment by luoping-12345** [comment]
LGTM

**Comment by dengyali0125** [comment]
LGTM

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplatechange/ServiceTemplateHistoryVo.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+
+package org.eclipse.xpanse.modules.models.servicetemplatechange;
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplatechange/ServiceTemplateHistoryVo.java:7]: how about moving 'change' folder insider 'servicetemplate' folder?

File: modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java [hunk 2]
```diff
@@ -485,10 +473,76 @@ public String getOpenApiUrl(UUID id) {
...
+                    .filter(history -> history.getRequestType() == requestType)
+                    .toList();
+        }
+        if (Objects.nonNull(changeStatus)) {
+            historyList = historyList.stream()
+                    .filter(history -> history.getStatus() == changeStatus)
+                    .toList();
+        }
+        return historyList.stream()
+                .sorted(Comparator.comparing(ServiceTemplateHistoryEntity::getCreateTime)
```
Comment (swaroopar) [inline modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java:502]: this is good. can we also write this in the API doc - the history returned is sorted by the ascending order of the change requested time.

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**